src/main/kotlin/solve/scene/view/LineView.kt

```diff
@@ -32,22 +33,18 @@ class LineView(
         node.endY = finishCoordinates.second
     }
 
-    private var width = OrdinaryWidth
-
     private fun createShape(): Line {
         val shape =
             Line(startCoordinates.first, startCoordinates.second, finishCoordinates.first, finishCoordinates.second)
-        shape.fill = line.layerSettings.getColor(line)
+        shape.stroke = line.layerSettings.color
```

Comment:
color manager is not used?

Reply:
Color manager is used when we set different colors for landmarks in one layer